api/src/main/java/org/bahmni/module/events/model/Event.java

```diff
@@ -0,0 +1,28 @@
+package org.bahmni.module.events.model;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class Event {
```

Comment:
Do you need to introduce a "Version" attribute, so that we know what version of Event Objects are travelling over the wire? This will be very helpful for schema changes and for client backward/forward compatibility.

Reply:
Right now i am not serializing / deserializing this also not saving to DB, once i start save it can we add that time the version attribute if that works ?

Reply:
The version attribute is for the client code (consumer). That doesn't have anything to do with DB save. I didn't understand.

Reply:
Right now we are not sending entire event in the payload. We are only sending the patient data and not the entire event

Reply:
You are creating an event. That event is a wrapper over OpenMRS Patient Object string. Doesn't your event need a version for client to know which event version their code is currently "reading". Please introduce a hardcoded versoin attributen in your wrapper event called: "1". And that will be updated to "2" when there is a breaking change.

Reply:
Added version attribute in the event class